electronics/_kicad_library/Polarity.pretty/Polarity_Center_Positive_2mm_SilkScreen.kicad_mod

```diff
@@ -0,0 +1,169 @@
+(module Polarity_Center_Positive_2mm_SilkScreen (layer F.Cu) (tedit 0)

```

Comment:
In that case, I think Symbols_Polarity.pretty would make the most sense, since this would indicate that its contents are similar to the original Symbols.pretty (as an addendum) and would appear next to it in the alphabetical list.